specs/interop/predeploys.md

```diff
@@ -420,6 +432,187 @@ dependency set called `dependencySet()`. This function MUST return the array of
 `L1Block` MUST also provide a public getter to get the dependency set size called `dependencySetSize()`. This function
 MUST return the length of the dependency set array.
 
+## OptimismMintableERC20Factory
+
+| Constant | Value                                        |
+| -------- | -------------------------------------------- |
+| Address  | `0x4200000000000000000000000000000000000012` |
+
+### OptimismMintableERC20
+
+The `OptimismMintableERC20Factory` creates ERC20 contracts on L2 that can be used to deposit
+native L1 tokens into (`OptimismMintableERC20`). Anyone can deploy `OptimismMintableERC20` contracts.
+
+Each `OptimismMintableERC20` contract created by the `OptimismMintableERC20Factory`
+allows for the `L2StandardBridge` to mint
+and burn tokens, depending on whether the user is
+depositing from L1 to L2 or withdrawing from L2 to L1.
+
+### Updates
+
+The `OptimismMintableERC20Factory` is updated to
+
+- use `CREATE3` instead of `CREATE2` to remove the compiler's dependency on address computation.
+- include a `deployments` mapping
+  to store the `remoteToken` address for each deployed `OptimismMintableERC20`.
+  - This is essential for the liquidity migration process defined in the liquidity migration spec.
+
+### Functions
+
+#### `createWithCreate3`
```

Comment:
This leaks an implementation detail into the name, not a fan of this style of naming